README.md

```diff
@@ -137,7 +137,7 @@ Optional parameters include:
 
 - `practitioner`: practitioner for which the measure will be calculated
 
-Currently, `measure` and `lastReceivedOn` parameters are not supported by the test server. The `subject-list` `reportType` is not supported by the test server - only `individual` and `population` `reportTypes` are supported at this time.
+Currently, `measure` and `lastReceivedOn` parameters are not supported by the test server. The `subject-list` `reportType` is not supported by the test server - only `subject` and `population` `reportTypes` are supported at this time.
```

Comment:
Because subject is not an actual MeasureReport type, should we specify here that the `subject` parameter corresponds with an individual measure report?